aim/aim_manager.py

```diff
@@ -150,6 +151,21 @@ def find(self, context, resource_class, **kwargs):
                 self._make_resource(context.db_session, resource_class, obj))
         return result
 
+    def get_status(self, context, resource):
```

Comment:
OK, thanks. Unless things change, my code will always be getting the DN and the status at the same time, but I understand the decision to separate these.